src/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -733,8 +733,11 @@ public with sharing class ADDR_Addresses_UTIL {
     public static String getParentAccsWithChildrenQuery() {
         String query = 'select recordTypeId, Current_Address__c, BillingStreet, BillingCity, BillingState, BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude';     
         if (ADDR_Addresses_UTIL.isStateCountryPicklistsEnabled)
-            query += ', BillingCountryCode, BillingStateCode ';
-        query += ', (select Current_Address__c, is_Address_Override__c, Primary_Address_Type__c, MailingStreet, MailingCity, MailingState, MailingPostalCode, MailingCountry, MailingLatitude, MailingLongitude from Account.Contacts)';  
+            query += ', BillingCountryCode, BillingStateCode';
```

Comment:
@kbromer This is where the issue with state/country picklists was fixed. (See also line 739 below.)